switch_model/wecc/get_inputs.py

```diff
@@ -17,14 +17,10 @@
 import os
 import sys
 import time
-
```

Comment:
This blank line denotes a separate between built-in python libraries and 3rd party python libraries that need to be installed. Common convention goes:
```
# built in libs
import os

# 3rd party libs
import sshtunnel

# modules from this project
import switch_model
```